hw3/02-charts.js

```diff
@@ -29,24 +29,80 @@ const borderColors = [
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-const renderChart = () => {
+const getColors = function getRepeatedColors(dataArray, colorArray) {
+  const repeatedColors = [];
+  dataArray.forEach((item, index) => {
+    repeatedColors.push(colorArray[index % colorArray.length]);
+  });
```

Comment:
What is "item" being used for?

![Screenshot (272)](https://github.com/srimel/frontend-homework/assets/38127011/0279b53e-3a9f-4369-9f00-442b9196de77)